### ISSUE TITLE
Fix vertical alignment of top navbar icon + text

### ISSUE THREAD

**Issue by taramk** [treehousedev/treehouse#87]
I added a hacky fix to align it in Safari (top 4px margin) but this made it misaligned in Chrome. Probably just need to use flexbox.

**Comment by taramk** [comment]
Removed the hacky fix, but now possibly slightly misaligned in Safari. Fine in Chrome last I checked.

**Comment by progrium** [comment]
@taramk can you include screenshots?

**Comment by taramk** [comment]
![Image](https://github.com/treehousedev/treehouse/assets/1813419/7839cd67-0601-466e-a38a-d5daf4af4b30)

kinda looks like it's just 1-2px off, but Search seems to be vertically aligned properly. this only happens in Safari.

**Comment by ArcaTech** [comment]
This one is weird. It looks like the SVGs are rendering slightly too low and the text (Today, Quick Add) slightly too high. I tried adding all the flexbox centering attributes (justify-content, align-items, align-self, etc) but it looks a lot like a Safari bug.

I spent a few hours troubleshooting but I'm not sure it's worth holding up other tickets for so I'm gonna do some of the others and then come back to this.
